DependencyInjection/DoctrineExtension.php

```diff
@@ -304,10 +313,22 @@ protected function getConnectionOptions($connection)
                 $options['global'][$key] = $value;
                 unset($options[$key]);
             }
+            if (!isset($options['global']['driver'])) {
+                $options['global']['driver'] = $options['driver'];
+            }
+            foreach ($options['shards'] as $i => $shard) {
+                if (!isset($shard['driver'])) {
+                    $options['shards'][$i]['driver'] = $options['driver'];
```

Comment:
Cause it will throw an exception to require `driver` or `driverOptions` in connection